apps/data-scanner/lib/models/session/Session.mjs

```diff
@@ -0,0 +1,56 @@
+import { Client } from './Client.mjs';
+import { writeToFile, readDir, readSqlite } from '../../utils/index.mjs';
+
+export class Session {
```

Comment:
I suggest rename Session to Pool or SessionPool or at least Sessions

Reply:
done